styling-and-appearance/themes-suite/common-styling-appearance-fluent-theme.md

```diff
@@ -55,6 +56,33 @@ The default values of the brushes in the theme are listed below:
 #### __Figure 1: Theme colors represented in RadOutlookBar__
 ![Fluent theme colors represented in RadOutlookBar](images/fluent-theme-outlook.PNG)	
 
+## FluentPalette Brushes
+
+### **Transparent Brushes** 
+
+* **PrimaryBrush**: Used in backgrounds of inputs.
+* **MouseOver**: Used for an interaction of buttons.
+* **PressedBrush**: Used for interaction of button-like controls.
+* **BasicBrush**: Used in the majority of the normal state borders.
+* **MainBrush**: Used for backgrounds of buttons, headers and footers.
+* **IconBrush**: Used for glyph and path icons, which should be 80% of black/white and for the light/dark variation respectfully
+
+### **Opaque Brushes**
+
+* **PrimaryBackgroundBrush**: Used when backgrounds need to be solid white/black
+* **AlternativeBrush**: Used for backgrounds of popups, windows, listboxes etc and is slightly darker/lighter.
+* **Complementary**: Slightly darker/lighter than the alternative in case of collisions
+* **AccentBrush**: Used for coloring the primary accent elements.
+* **AccentMouseOverBrush**: Variation of the __AccentBrush__ that is used for the hovered states of primary accent elements.
+* **AccentPressedBrush**: Variation of the __AccentBrush__ that is used for the pressed states of primary accent elements.
+* **AccentFocusedBrush**: Variation of the __AccentBrush__ that is used for the focused states of primary accent elements.
+* **MarkerBrush**: Used for all foregrounds which are black/white.
+* **MarkerInvertedBrush**: Used for interactions states which is white/white.
```

Comment:
Used for the foregrounds in interaction or accent states - white in both variations.